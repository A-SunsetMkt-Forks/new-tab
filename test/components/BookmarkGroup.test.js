'use strict';

const BookmarkGroup = require('../../src/components/BookmarkGroup.html');

describe('BookmarkGroup component', () => {
  it('renders correctly', () => {
    const target = document.createElement('div');
    new BookmarkGroup({ target });
    expect(target.innerHTML).toMatchSnapshot();
  });
});
