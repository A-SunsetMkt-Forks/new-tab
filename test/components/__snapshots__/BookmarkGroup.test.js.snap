// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookmarkGroup component renders correctly 1`] = `
"<div id=\\"bookmarks\\">

  

  </div>"
`;
